config/locales/zgh.yml

```diff
@@ -89,9 +89,9 @@ zgh:
     '404': The page you are looking for isn't here.
     '406': This page is not available in the requested format.
     '410': The page you were looking for doesn't exist here anymore.
-    '422': 
+    '422':
```

Comment:
Not a criticism but that's an awful lot of files changed for a setting change 😜

Reply:
@CEbbinghaus The linter and localisation tools are acting strangely on my laptop. I would have fixed by now but I got the indication that this PR wasn't going to be merged into Mastodon. Let me give it another shot.